manualdirectdebit.php

```diff
@@ -218,3 +218,14 @@ function isCustomGroupDirectDebit($groupID) {
 
   return $groupID == $directDebitMandateId;
 }
+
+/**
+ * Checks if operation is acceptable
+ *
+ * @param $op
+ *
+ * @return bool
+ */
+function isAcceptableOperation($op) {
```

Comment:
please remove this function and use ( $op == 'create' || $op == 'edit') directly.